renderer/components/TrackTimeModal/TrackTimeModal.tsx

```diff
@@ -131,6 +132,16 @@ export default function TrackTimeModal({
 
   useEffect(() => {
     addSuggestions(activities, latestProjAndDesc, latestProjAndAct);
+    const tempProj = Object.keys(latestProjAndAct);
+    if (localStorage.getItem("year-projects")) {
+      const yearProj = localStorage.getItem("year-projects").split(",");
```

Comment:
let's split variables and conditions with empty string
Not only in this place. Everywhere when you see it.
It boosts code readability